### PR TITLE
Updated to fix grammatical and punctuation issues

### PR DESCRIPTION
**Pull request by nehbehl**
feat: Grammatical and punctuation updates

1. Added missing quote in the second instance of close at line number 88
2. Corrected walkthrough as it is a single word
3. Updated line number 152 for grammatical mistake
4. Short-form IE
5. pytest corrected

### CODE REVIEW

File: source/getting-started.rst [hunk 1]
```diff
@@ -96,7 +96,7 @@ Using Selenium to write tests
 
 Selenium is mostly used for writing test cases.  The `selenium` package itself
 doesn't provide a testing tool/framework.  You can write test cases using
-Python's unittest module.  The other options for a tool/framework are py.test
+Python's unittest module.  The other options for a tool/framework are pytest
 and nose.
```
Comment (baijum) [inline source/getting-started.rst:100]: Can you please add links to pytest & nose websites here?
- https://docs.pytest.org/en/stable/
- https://nose.readthedocs.io/en/latest/

File: source/getting-started.rst [hunk 2]
```diff
@@ -141,15 +141,15 @@ You can run the above test case from a shell like this::
 The above result shows that the test has been successfully completed.
 
 
-Walk through of the example
+Walkthrough of the example
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (baijum) [inline source/getting-started.rst:145]: Please remove one character from underline (`~`)

**Comment by baijum** [comment]
Thanks for carefully reading the documentation and trying to make it perfect!

I have added a couple of comments, please take a look.

File: source/getting-started.rst (hunk 1 above)
Comment (nehbehl) [reply source/getting-started.rst:100]: Sure

File: source/getting-started.rst (hunk 2 above)
Comment (nehbehl) [reply source/getting-started.rst]: Done.

File: source/getting-started.rst (hunk 1 above)
Comment (baijum) [reply source/getting-started.rst:100]:
```suggestion
Python's unittest module.  The other options for a tool/framework are `pytest <https://docs.pytest.org/en/stable/>`_
and `nose <https://nose.readthedocs.io/en/latest/>`_.
```

File: source/getting-started.rst [hunk 3]
```diff
@@ -96,8 +96,10 @@ Using Selenium to write tests
 
 Selenium is mostly used for writing test cases.  The `selenium` package itself
 doesn't provide a testing tool/framework.  You can write test cases using
-Python's unittest module.  The other options for a tool/framework are py.test
-and nose.
+Python's unittest module.  The other options for a tool/framework are `pytest <https://docs.pytest.org/en/stable/>`_
+and `nose <https://nose.readthedocs.io/en/latest/>`_.
+- https://docs.pytest.org/en/stable/
+- https://nose.readthedocs.io/en/latest/
```
Comment (baijum) [inline source/getting-started.rst:102]:
```suggestion
```

**Comment by baijum** [comment]
Thanks @nehbehl :+1:

**Pull request merged**